### DISCUSSION TITLE
Machine-wide (common for all users) config on Windows?

### DISCUSSION THREAD

**Discussion by nyq** [rustdesk/rustdesk#5765] [Q&A] [answered]
On Windows, RustDesk seems to store configuration files under `C:\Users\<username>\AppData\Roaming\RustDesk\config\`, which makes it user-specific configuration. Is there a way to have one machine-wide configuration for all users that would override user-specific configuration?

**Comment by dinger1986** [comment, marked as answer]
There is, it's stored in serviceprofiles

**Reply by nyq** [reply]
Thank you, this is very useful!
